test/demo/createCenter.inttest.ts

```diff
@@ -45,23 +46,20 @@ describe('When creating a notification with the center showing', () => {
     describe('When passing a valid set of options', () => {
         const options: NotificationOptions = {
             body: 'Test Notification Body',
-            title: 'Test Notificaiton Title',
+            title: 'Test Notification Title',
             id: 'test-notification-0',
             icon: 'https://openfin.co/favicon.ico'
         };
 
         let createPromise: Promise<Notification>;
         let note: Notification;
         beforeEach(async () => {
-            createPromise = notifsRemote.create(testWindow.identity, options);
-
-            // We want to be sure the operation is completed, but don't care if it succeeds
-            note = await createPromise.catch(() => ({} as Notification));
+            ({createPromise, note} = await notifsRemote.createAndAwait(testWindow.identity, options));
```

Comment:
I feel like there's something going on here I'll need explained to me. I don't understand why these calls are all wrapped in brackets

Reply:
JS syntax quirk... Without the brackets, the braces are interpreted as being a code block.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Assignment_without_declaration

Reply:
As Phil said it's a destructured assignment. A bit weird I agree, but probably the best way to do it for now.